src/main/java/net/pms/network/mediaserver/jupnp/support/contentdirectory/UmsContentDirectoryService.java

```diff
@@ -423,7 +423,7 @@ private void checkInput(Result modelObjectToAdd) {
 		if (modelObjectToAdd.getContainers().size() > 1) {
 			LOGGER.trace("more than 1 container ... using first found.");
 		}
-		if (modelObjectToAdd.getContainers().size() > 0 && modelObjectToAdd.getItems().size() > 0) {
+		if (modelObjectToAdd.getContainers().isEmpty() && modelObjectToAdd.getItems().isEmpty()) {
```

Comment:
@SurfaceS is it correct? What about to use 

    if (!modelObjectToAdd.getContainers().isEmpty() && !modelObjectToAdd.getItems().isEmpty()) {